rotkehlchen/tests/api/test_exchanges.py

```diff
@@ -736,212 +733,6 @@ def assert_okay(response: 'Response') -> None:
     assert len(result['entries']) == 0
 
 
-@pytest.mark.parametrize('number_of_eth_accounts', [0])
-@pytest.mark.parametrize('added_exchanges', [(Location.KRAKEN, Location.POLONIEX)])
-@pytest.mark.parametrize('start_with_valid_premium', [False, True])
-def test_query_asset_movements(
```

Comment:
There is no other test to write instead of this? Just removing it as the endpoint no longer exists right?

Reply:
Yes, removing this since the endpoint is being removed. The stuff like order by, time range, pagination, etc that was being tested here should already be in place and properly tested for history events, so I don't think any new tests need made for the new history event based asset movements.